src/client/sync-client.ts

```diff
@@ -43,10 +43,11 @@ export function makeSyncClient(opts: CommonOptions = {}): SyncClient {
 
       for (const syncServer of syncServers) {
         const url = `${syncServer}/api/v2/store/${syncKey}`
+        const cleanUrl = url.replace(syncKey, 'xxxx')
```

Comment:
This is very nice! 👍 Only suggestion is to use the "numb" nomenclature. "Clean" is appropriate enough, but I feel like "numb" is even more so (something that is no longer sensitive is _numb_). Please change to `numbUrl`.